packages/host/tests/acceptance/operator-mode-acceptance-test.gts

```diff
@@ -980,14 +980,14 @@ module('Acceptance | operator mode tests', function (hooks) {
       assert.dom('[data-test-profile-popover]').doesNotExist();
       assert.dom('[data-test-settings-modal]').exists();
 
-      assert.dom('[data-test-profile-icon]').hasText('T'); // "T", from first letter of: @testuser:staging
+      assert.dom('[data-test-profile-icon]').hasText('T'); // "T", from first letter of: @testuser:localhost
       assert.dom('[data-test-profile-display-name]').hasText(''); // No display name set yet
       assert
         .dom('[data-test-profile-icon]')
-        .hasStyle({ backgroundColor: 'rgb(34, 221, 152)' });
+        .hasStyle({ backgroundColor: 'rgb(88, 226, 29)' });
```

Comment:
what happened here?

Reply:
The "server" of the testuser changed from staging to localhost, which results in a different color for the avatar background